.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.7
+      - image: neo4j:3.1.0-enterprise
+        environment:
+          NEO4J_AUTH: none
+          NEO4J_HEAP_MEMORY: 256
+          NEO4J_CACHE_MEMORY: 256M
+
+    working_directory: /go/src/github.com/Financial-Times/people-rw-neo4j
+    environment:
+      CIRCLE_TEST_REPORTS: /tmp/test-reports
+      CIRCLE_ARTIFACTS: /tmp/artifacts
+      NEO4J_TEST_URL: "http://localhost:7474/db/data/"
+
+    steps:
+      - checkout
+      - run:
+          name: "Get build dependencies"
+          # Not using govendor for syncing deps as vendoring is not yet introduced for this repository
+          command: |
+            go get -u github.com/kardianos/govendor
+            # govendor sync
+            go get -t -d -v ./...
+      - run:
+          name: "Build project"
+          command: |
+            go build -v
+      - run:
+          name: "Get tests dependencies"
+          command: |
+            go get -u github.com/jstemmer/go-junit-report
+            go get -u github.com/mattn/goveralls
+            wget https://raw.githubusercontent.com/Financial-Times/cookiecutter-upp-golang/master/coverage.sh && chmod +x coverage.sh
+      - run:
+          name: "Wait for Neo4j to start up"
+          command: |
+            wget --retry-connrefused --no-check-certificate -T 60 $NEO4J_TEST_URL; curl $NEO4J_TEST_URL
+
+      - run:
+          name: "Run tests"
+          command: |
+            mkdir -p $CIRCLE_TEST_REPORTS/golang
+            mkdir -p $CIRCLE_ARTIFACTS
+            govendor test -race -v +local | go-junit-report > $CIRCLE_TEST_REPORTS/golang/junit.xml
+            ./coverage.sh
+      - run:
+          name: "Compute coverage"
+          command: |
+            goveralls -coverprofile=$CIRCLE_ARTIFACTS/coverage.out -service=circle-ci -repotoken=$COVERALLS_TOKEN
+
+      - store_test_results:
+          path: /tmp/test-reports
+
+      - store_artifacts:
+          path: /tmp/artifacts
+          destination: build
```

Comment:
We can also to a build of the docker image:

  docker_build:
    working_directory: /public-concordances-api
    docker:
      - image: docker:1.12.6-git
    steps:
      - checkout
      - setup_docker_engine
      - run:
          name: Build Dockerfile
          command: docker build .